.github/workflows/publish-operator-bundle.yaml

```diff
@@ -0,0 +1,74 @@
+name: "Publish operator bundle"
+
+on:
+  push:
+    paths:
+      - '.github/workflows/publish-operator-bundle.yaml'
+      - 'bundle.Dockerfile'
+    branches:
+      - main
+    tags:
+      - 'v*'
+  workflow_dispatch:
+
+jobs:
+  publish:
+    runs-on: ubuntu-20.04
+
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Docker meta
+        id: docker_meta
+        uses: docker/metadata-action@v4
+        with:
+          images: |
+            otel/opentelemetry-operator-bundle
+            ghcr.io/open-telemetry/opentelemetry-operator/opentelemetry-operator-bundle
+          tags: |
+            type=semver,pattern={{version}}
+            type=semver,pattern={{major}}.{{minor}}
+            type=semver,pattern={{raw}}
+            type=ref,event=branch
+
+
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v2
+
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v2
+
+      - name: Cache Docker layers
+        uses: actions/cache@v3
+        with:
+          path: /tmp/.buildx-cache
+          key: ${{ runner.os }}-buildx-${{ github.sha }}
+          restore-keys: |
+            ${{ runner.os }}-buildx-
+
+      - name: Log into Docker.io
+        uses: docker/login-action@v2
+        if: ${{ github.event_name == 'push' }}
+        with:
+          username: ${{ secrets.DOCKER_USERNAME }}
+          password: ${{ secrets.DOCKER_PASSWORD }}
+
+      - name: Login to GitHub Package Registry
+        uses: docker/login-action@v2
+        if: ${{ github.event_name == 'push' }}
```

Comment:
Why is this only for `push`, but not for the `workflow_dispatch` event?

Reply:
Same as above